signals/sender/sender_is_known_boa_domain.yml

```diff
@@ -0,0 +1,4 @@
+name: "Sender: Known Bank of America domain"
+type: "query"
+source: |
```

Comment:
May want to add a DMARC check here, unless risk score will weigh a DMARC failure more/a failure would be accounted for elsewhere.